portality/scripts/articles_identical_issns.py

```diff
@@ -0,0 +1,42 @@
+from portality import models
+from portality.core import es_connection
+from portality.util import ipt_prefix
+import esprit
+import re
+import csv
+
+ALL = {
+    "query": {
+        "match_all": {}
+    }
+}
+
+
+if __name__ == "__main__":
+
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--out", help="output file path")
+    args = parser.parse_args()
+
+    if not args.out:
+        print("Please specify an output file path with the -o option")
```

Comment:
if the script no longer requires an output file specified, add it as a default in `add_argument` and don't ask the user for it when it's missing

Reply:
oh no, it's other way round. I forgot to remove the "out.csv" output that I used for debugging and uncomment the exit.